components/filter/filter.js

```diff
@@ -292,29 +303,35 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 	}
 
 	_handleDimensionShowComplete() {
-		const dimension = this.shadowRoot.querySelector(`[data-key="${this._activeDimensionKey}"]`);
-		const focusable = getFirstFocusableDescendant(dimension);
-		if (focusable) {
-			focusable.focus();
-		}
+		const returnButton = this.shadowRoot.getElementById('d2l-filter-dimension-header-return');
```

Comment:
Would `this.shadowRoot.querySelector('d2l-button-icon')` work as well? Just avoids an `id`.

Reply:
Yeah that _should_ work - the clear buttons won't be icon buttons, and I can narrow it down to the header

Reply:
Or even `.querySelector('d2l-button-icon[icon="tier1:chevron-left"]')`.